jwst/resample/resample_step.py

```diff
@@ -93,6 +93,13 @@ def process(self, input):
         else:
             result = resamp.output_models
 
+        # remove irrelevant WCS keywords
+        rm_keys = ['v2_ref', 'v3_ref', 'ra_ref', 'dec_ref', 'roll_ref',
+                   'vi_yang', 'vparity']
```

Comment:
vi_yang should be "v3yangle"  (see https://github.com/spacetelescope/jwst/blob/master/jwst/datamodels/schemas/wcsinfo.schema.yaml#L301)